config/application.rb

```diff
@@ -107,7 +107,9 @@ class Application < Rails::Application
 
     # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    config.time_zone = ENV["TIMEZONE"]
+    #
+    # Be sure to set the appropriate locale in application.yml
+    config.time_zone = ENV.fetch("TIMEZONE", "UTC")
```

Comment:
There is one little caveat. Our specs depend on Melbourne time. We could just replace `UTC` with `Melbourne` here, but the better solution is probably to set `config.time_zone = "Melbourne"` in `config/environments/test.rb`. Can you add that and check that `spec/controllers/admin/order_cycles_controller_spec.rb` is still passing?

Reply:
Ok, you're right, that test was broken.

Changed the approach :+1:, since now all three environments have a different approach (development can use whatever, test needs `Melbourne`, and in production/staging we want to enforce setting it in `application.yml`).